code/modules/projectiles/guns/energy/stun.dm

```diff
@@ -6,16 +6,44 @@
 	ammo_type = list(/obj/item/ammo_casing/energy/electrode)
 	ammo_x_offset = 3
 
-/obj/item/weapon/gun/energy/stunrevolver
-	name = "stun revolver"
+/obj/item/weapon/gun/energy/shock_revolver
+	name = "shock revolver"

```

Comment:
So? This is nominally being named by in-universe spessmens, and there are plenty of things that were named for something they resemble without being the same.